suite-native/navigation/src/useStackNavigation.ts

```diff
@@ -0,0 +1,26 @@
+import { ParamListBase, useNavigation } from '@react-navigation/native';
+
+import { StackToTabCompositeProps } from './index';
+
+type NavigationProp = StackToTabCompositeProps<ParamListBase, string, ParamListBase>;
+
+export const useStackNavigation = () => {
```

Comment:
I would name this more specifically since it's doing only one thing and not handling whole navigation

Reply:
What's your suggestion? `useInitialScreenNavigation` maybe? 🤔

Reply:
Renamed to `useNavigateToInitialScreen`.